setup.py

```diff
@@ -79,7 +79,14 @@ def filter_sources(f):
         "Operating System :: Unix",
     ],
     packages=find_packages(),
-    ext_modules=ext_modules,
-    cmdclass={"build_ext": BuildExtension},
-    install_requires=["absl-py", "numpy", "torch"],
+    # ext_modules=ext_modules,
```

Comment:
Do we not need these two lines that are commented out? If so, could you explain and also remove the lines from the file.

Reply:
We no longer need Sputnik build -- so should we remove Sputnik related installation code in `setup.py`.

Reply:
Great, can you go ahead and do that?